File: 8-kyu/a-wolf-in-sheep's-clothing/swift/solution.swift
func warnTheSheep(_ queue: [String]) -> String {
    let wolfIdx = queue.firstIndex(of: "wolf")!
    return wolfIdx == queue.count - 1 ? "Pls go away and stop eating my sheep" : "Oi! Sheep number \(queue.count - wolfIdx - 1)! You are about to be eaten by a wolf!"
}

File: 8-kyu/a-wolf-in-sheep's-clothing/javascript/solution.js
const warnTheSheep = (queue) => {
  const wolfIdx = queue.indexOf("wolf");
  return wolfIdx === queue.length - 1 ? "Pls go away and stop eating my sheep" : `Oi! Sheep number ${queue.length - wolfIdx - 1}! You are about to be eaten by a wolf!`;
}